pkg/components/microservice.go

```diff
@@ -129,6 +133,7 @@ func (m *microserviceImpl) rebuildDeployment() *appsv1.Deployment {
 			Image: m.image,
 		},
 	}
+	m.builtDeployment.Spec.Template.Spec.Tolerations = m.tolerations
```

Comment:
Aren't they already being set in `deployment.Build()`?

Reply:
Yes, looks like you right, overengineered that a bit =)